lib/classes/Variables.js

```diff
@@ -661,7 +661,6 @@ class Variables {
 
     // Process JS files
     if (fileExtension === 'js') {
-      // eslint-disable-next-line global-require, import/no-dynamic-require
```

Comment:
This should not be part of this PR

Reply:
will undo this as this was change after adding `.eslintrc.js`